.common-ci.yml

```diff
@@ -81,6 +81,12 @@ trigger-pipeline:
     matrix:
       - DRIVER_VERSION: [535.183.06, 550.90.12, 560.35.03]
 
+# Define the driver versions for jobs that can be run in parallel for ubuntu24.04
+.driver-versions-ubuntu24.04:
```

Comment:
Note, since all active datacenter branches are supported for ubuntu22.04 and rhel9, we can remove the matrix definition for those OS versions.

Reply:
This is done, thanks!